.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19.1
+          go-version: 1.20.4
```

Comment:
We don't actually use Go here right now, do we?

Reply:
Oh, but we will with the matrix!

Reply:
Thanks for the catch; probably a leftover from before the containerization 👍 

Note I'll merge the two tests.yml, which means we'll need go to call:

```yaml
- run: go install github.com/ipfs/kubo/cmd/ipfs@${{ matrix.target }}
```

In place of

```yaml
# only work with released binaries
- uses: ipfs/download-ipfs-distribution-action@v1
```